src/validation/schema.js

```diff
@@ -389,6 +389,282 @@ const validateNoImportDirective = def =>
+        message: `fulltext include argument must have two fields, 'entity' and 'fields'}`,
+      }),
+    ])
+  }
+  return argument.fields.reduce(
+    (errors, field) =>
+      errors.concat(
+        validateFulltextDirectiveArgumentIncludeFields(def, directive, field),
+      ),
+    List([]),
+  )
+}
+
+const validateFulltextDirectiveArgumentIncludeFields = (def, directive, field) => {
+  if (!['entity', 'fields'].includes(field.name.value)) {
+    return List().push(
+      immutable.fromJS({
+        loc: directive.name.loc,
+        entity: def.name.value,
+        message: `@fulltext field '${field.name.value}' invalid, may only be one of: [entity, fields]`,
+      }),
+    )
+  }
+  if (field.name.value == 'entity' && field.value.kind != 'StringValue') {
+    return List().push(
+      immutable.fromJS({
+        loc: directive.name.loc,
+        entity: def.name.value,
+        message: `@fulltext include field 'entity' must be a string`,
+      }),
+    )
+  } else if (field.name.value == 'fields' && field.value.kind != 'ListValue') {
+    return List().push(
+      immutable.fromJS({
+        loc: directive.name.loc,
+        entity: def.name.value,
+        message: `@fulltext entity 'fields' must be Lists`,
+      }),
+    )
+  } else if (field.name.value == 'fields' && field.value.kind == 'ListValue') {
+    return field.value.values.reduce(
+      (errors, field) =>
+        errors.concat(
+          validateFulltextDirectiveArgumentIncludeArgumentFieldsObjects(
+            def,
+            directive,
+            field,
+          ),
+        ),
+      List([]),
+    )
+  } else {
+    return List([])
+  }
+}
+
+const validateFulltextDirectiveArgumentIncludeArgumentFieldsObjects = (
+  def,
+  directive,
+  argument,
+) => {
+  if (argument.kind != 'ObjectValue') {
+    return List().push(
+      immutable.fromJS({
+        loc: directive.name.loc,
+        entity: def.name.value,
+        message: `@fulltext argument 'include' entity fields items must be objects`,
+      }),
+    )
+  } else {
+    return argument.fields.reduce(
+      (errors, field) =>
+        errors.concat(
+          validateFulltextDirectiveArgumentIncludeArgumentFieldsObject(
+            def,
+            directive,
+            field,
+          ),
+        ),
+      List(),
+    )
+  }
+}
+
+const validateFulltextDirectiveArgumentIncludeArgumentFieldsObject = (
+  def,
+  directive,
+  field,
+) => {
+  if (!['name'].includes(field.name.value)) {
+    return List([]).push(
+      immutable.fromJS({
+        loc: directive.name.loc,
+        entity: def.name.value,
+        message: `@fulltext field '${field.name.value}' invalid, may only be one of: [name]`,
+      }),
+    )
+  } else if (field.name.value == 'name' && field.value.kind != 'StringValue') {
+    return List([]).push(
+      immutable.fromJS({
+        loc: directive.name.loc,
+        entity: def.name.value,
+        message: `@fulltext include field 'name' must be a string`,
```

Comment:
`@fulltext include field 'name' must be a string` -> `@fulltext argument 'include > entity > field > name' must be the name of an entity field`.

Reply:
This error is triggered when the provided value is not provided of kind `StringValue`, so I feel it would be important to remind them to format the provided value correctly with double quotes.  
How about?  `@fulltext argument 'include > entity > field > name' must be the name of an entity field enclosed in double quotes`